frontend/src/data/utils/datasetutils.ts

```diff
@@ -178,6 +178,41 @@ export function shouldShowAltPopCompare(fromProps: ShouldShowAltPopCompareI) {
   return (
     fromProps.fips.isState() &&
     fromProps.breakdownVar === RACE &&
-    fromProps.variableConfig.variableId === COVID_VAXX
+    fromProps.variableConfig.variableId === "covid_vaccinations"
```

Comment:
Is this a regression?

Reply:
No, I eliminated that CONST, since the only reason we were using so much was the previous hard-coding of UI suppression for COVID VAXX at county level.